src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -741,12 +771,24 @@ protected MaxwellOptionParser buildOptionParser() {
 				.withRequiredArg();
 		parser.separator();
 
-		parser.accepts( "ha", "enable high-availability mode via jgroups-raft" )
+		parser.accepts( "ha", "enable high-availability mode via jgroups-raft or zookeeper" )
 				.withOptionalArg().ofType(Boolean.class);
 		parser.accepts( "jgroups_config", "location of jgroups xml configuration file" )
 				.withRequiredArg();
 		parser.accepts( "raft_member_id", "raft memberID.  (may also be specified in raft.xml)" )
 				.withRequiredArg();
+		parser.accepts("zookeeper_server","enable maxwell High Availability using zookeeper")
+				.withRequiredArg();
+		parser.accepts("session_time_out_ms","session timeout duration (maxwellHA on zk)")
+				.withRequiredArg().ofType(Integer.class);
+		parser.accepts("connection_time_out_ms","connection timeout duration (maxwellHA on zk)")
+				.withRequiredArg().ofType(Integer.class);
+		parser.accepts("max_retries","maximum retry (maxwellHA on zk)")
+				.withRequiredArg().ofType(Integer.class);
+		parser.accepts("base_sleep_time_ms","initial retry wait time (maxwellHA on zk)")
+				.withRequiredArg().ofType(Integer.class);
+		parser.accepts("ha_modality","high availability mode : jgroups-raft or zookeeper")
+				.withRequiredArg();
```

Comment:
instead of introducing a new parameter, let's make the `ha` parameter accept and optional server string.  So:

`--ha` still enables HA via raft, but `--ha=zookeeper` enables HA via ZK.